frontend/www/js/omegaup/arena/location.ts

```diff
@@ -42,7 +44,8 @@ export function getOptionsFromLocation(location: string): LocationOptions {
         response.popupDisplayed = PopupDisplayed.RunSubmit;
       } else if (match.groups.popup?.startsWith('show-run')) {
         response.guid = match.groups.popup.split(':')[1];
-        response.popupDisplayed = PopupDisplayed.RunDetails;
+        (response.problemAlias = response.problem.problem.alias),
+          (response.popupDisplayed = PopupDisplayed.RunDetails);
```

Comment:
el uso del operador coma es intencional?

```suggestion
        response.problemAlias = response.problem.problem.alias;
        response.popupDisplayed = PopupDisplayed.RunDetails;
```

(_creo_ que esto lo está insertando Prettier cuando una sentencia no termina en `;`)

Reply:
Ahhh, si fue el prettier. Me confié en que autocompletaba los `;`, pero ahora tendré más cuidado en eso.